### DISCUSSION TITLE
Wiki Pages Does not show up anywhere!

### DISCUSSION THREAD

**Discussion by StellarWriter** [community/community#52202] [New to GitHub]
Hi, I am trying to find wikipages, it does not show up where it is supposed to be and I cannot open it anywhere.
Anyone have any advice?

**Comment by SMelidoni** [comment]
Hi there @StellarWriter 👋🏽

Correct me if I'm wrong but It sounds like you're looking for the Wiki feature in your GitHub repository. To locate and access the Wiki, follow these steps:

1. **Navigate to the repository**: Open the GitHub repository where you expect the Wiki to be.

2. **Find the Wiki tab**: Look for the "Wiki" tab near the top of the repository page, alongside other tabs like "Code", "Issues", "Pull Requests", etc.

   If the "Wiki" tab is not visible, it could be disabled for your repository. To enable it:

   1. Click on the "Settings" tab in your repository.
   2. Scroll down to the "Features" section.
   3. Check the box next to "Wikis" to enable the Wiki feature.

3. **Access the Wiki**: Once the "Wiki" tab is visible, click on it to open the Wiki and view the pages.

If you've followed these steps and still cannot locate the Wiki, please provide more information about your repository, and I'll be happy to help you further!

**Reply by jeremiahr0430** [reply]
It asks for an upgrade for enabling it..

**Reply by samarpita-saha293** [reply]
Yes, and that upgrade options needs you to make a payment of $48.

**Reply by Will-Blair** [reply]
It's available in the free version if you have created the repo as a Public one:

![image](https://github.com/user-attachments/assets/46103e57-1761-4531-bad6-2e69dbdd1bc4)

If you created it as a Private one, then you have to pay for the upgrade.
